run.py

```diff
@@ -406,7 +448,7 @@ def numericalize(self, ds):
                 'is_truncated': 'True' if (not (attention_mask == 0).any() ) else 'False', #HACK has to be string so that its ignored by the model
             }
             if not self.no_ground_truth:
-                dct['label'] = labels[ind]
+                dct['labels'] = labels[ind]
```

Comment:
Not sure if it was changed to "labels" in a more recent version or if it is changed automatically from 'label' to 'labels' at some point, a bit strange...